platform/android/MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/testapp/geometry/GeoJsonConversionTest.java

```diff
@@ -22,10 +28,99 @@
 
   // Regression test for #12343
   @Test
-  public void testGetCameraForLatLngBounds() {
+  public void testEmptyFeatureCollection() {
+    validateTestSetup();
+    onMapView().perform(getMapboxMapAction((uiController, mapboxMap) -> {
+      mapboxMap.addSource(
+        new CustomGeometrySource("test-id",
+          new CustomProvider(fromFeatures(singletonList(fromGeometry(fromGeometries(emptyList())))))
+        )
+      );
+      mapboxMap.addLayer(new SymbolLayer("test-id", "test-id"));
+      assertNotNull(mapboxMap);
```

Comment:
Do we need these assertions in all the test cases?

Reply:
These are a remains of an initial test, removed with https://github.com/mapbox/mapbox-gl-native/pull/12354/commits/c1148a5fcd9edaae888f11ede2e0560cc21695e0